BitmovinComscoreAnalytics/Classes/ComScoreAnalytics.swift

```diff
@@ -28,7 +33,7 @@ public final class ComScoreAnalytics {
 
     public static func createComScoreStreamingAnalytics(bitmovinPlayer: BitmovinPlayer, metadata: ComScoreMetadata) throws -> ComScoreStreamingAnalytics? {
         if started {
-            return ComScoreStreamingAnalytics(bitmovinPlayer: bitmovinPlayer, metadata: metadata)
+            return ComScoreStreamingAnalytics(bitmovinPlayer: bitmovinPlayer, configuration: ComScoreAnalytics.configuration!, metadata: metadata)
```

Comment:
We shouldn't force unwrap here. It probably wont be needed if you just store the value instead of the whole configuration